### PR TITLE
Updated

### PR DESCRIPTION
**Pull request by 6406NathanVanZyl**

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -14,7 +14,7 @@ static void Main(string[] args)
             Console.ReadLine();
             Console.WriteLine("You can make it pal");
 Console.WriteLine("Hi its Trish");
-
+//Comment
```
Comment (TrishanaaParsard) [inline Program.cs:17]: Added commnet

**Review by TrishanaaParsard — DISMISSED** [review]

**Review by 6406NathanVanZyl — COMMENTED** [review]
decline

**Review by TrishanaaParsard — CHANGES_REQUESTED** [review]
Change code to Console.Writline("Alfred giving us instructions");

File: Program.cs [hunk 2]
```diff
@@ -14,10 +14,10 @@ static void Main(string[] args)
             Console.ReadLine();
             Console.WriteLine("You can make it pal");
 Console.WriteLine("Hi its Trish");
+//Comment
 
 
-
-
```
Comment (TrishanaaParsard) [inline Program.cs:20]: yeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeet

**Review by TrishanaaParsard — COMMENTED** [review]
.

**Pull request merged**